openedx/core/djangoapps/user_api/tests/test_helpers.py

```diff
@@ -147,6 +147,38 @@ def test_invalid_restriction(self):
         with self.assertRaises(InvalidFieldError):
             desc.add_field("name", field_type="text", restrictions={"invalid": 0})
 
+    def test_option_overwrite_passes_through(self):
+        desc = FormDescription("post", "/submit")
+        field = {
+            "name": "country",
+            "label": "Country",
+            "field_type": "select",
+            "default": "PK",
+            "required": True,
+            "error_messages": {
+                "required": "You must provide a value!"
+            },
+            "options": [
+                ("US", "United States of America"),
+                ("PK", "Pakistan")
+            ]
+        }
+        desc.override_field_properties(
+            field["name"],
+            options=[
```

Comment:
Instead of setting `options` in the `desc.override_field_properties` call, set `default='PK'`, and then pass the options (without default parameters) to the field constructor as you normally would - that's the path we're using.

In other words, this chunk should look like this:

```python
desc.override_field_properties(
    field['name'],
    default='PK'
)
```